lambdas/build-cri-oauth-request/src/test/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandlerTest.java

```diff
@@ -293,13 +302,19 @@ void shouldReceive200ResponseCodeAndReturnCredentialIssuerResponseWithoutRespons
                 .thenReturn(addressOauthCriConfig.getComponentId());
         when(mockIpvSessionService.getIpvSession(SESSION_ID)).thenReturn(mockIpvSessionItem);
         mockVcHelper.when(() -> VcHelper.isSuccessfulVc(any())).thenReturn(true, true);
-        when(userIdentityService.getIdentityCredentials(any()))
+        when(VcHelper.filterVCBasedOnProfileType(any(), any()))
                 .thenReturn(
                         List.of(
                                 generateVerifiableCredential(
```

Comment:
This generates the new VerifiableCredential class object, so wants to be passed a userId and criId